packages/dx-react-demos/src/material-ui/column-chooser/basic.jsx

```diff
@@ -68,7 +68,7 @@ export default class Demo extends React.PureComponent {
             </Toolbar>
             <ColumnChooser
               columns={columns}
-              hiddenColumns={hiddenColumns}
+              hiddenColumnNames={hiddenColumnNames}
               onHiddenColumnsChange={this.hiddenColumnsChange}
```

Comment:
Maybe we also rename onHiddenColumnsChange to onHiddenColumnNamesChange?

Reply:
It seems too long

Reply:
@viterobk

Reply:
Let's leave it as is.

Reply:
Why do we introduce this complexity in naming guidelines? There is a simple rule. The onXXXChange spelling for an event and defaultXXX for a starting value.